crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -1140,6 +1140,47 @@ def prepare_all_compute_nodes
         end
       end
 
+      # Restart remote resources at target node from cluster node
+      # This is needed for services (like nova-compute) managed by pacemaker.
+      def restart_remote_resources(controller, node)
+        hostname = node[:hostname]
+        out = controller.run_ssh_cmd(
+          "crm --wait node standby remote-#{hostname}",
```

Comment:
`crm resource restart` was my initial idea. But we would need to know the services, and it's not always the same set (e.g. with openvswitch we have remote services for that as well).

So I've asked yago from HA team and he got an idea that putting remote node into standby node and back restarts all remote services running there.